rubrik_cdm/rubrik_cdm.py

```diff
@@ -22,6 +22,7 @@
 import logging
 from random import choice
 import time
+import socket
```

Comment:
I know there is some reluctance to import additional libraries, but this was the best way to determine the scope (i.e. local network adapter) used to connect to the cluster. Without determining scope, the OS will not create the connection.